### PR TITLE
Update documentation on CPU monitor issue

### PR DESCRIPTION
**Pull request by kthui**

### CODE REVIEW

**Comment by kthui** [comment]
Merge only after https://github.com/triton-inference-server/model_analyzer/pull/189 is merged.

File: docs/metrics.md [hunk 1]
```diff
@@ -63,6 +63,8 @@ recorded and aggregated over fixed intervals during a perf analyzer run.
 * `cpu_used_ram`: The total amount of memory used by all CPUs
 * `cpu_available_ram`: The total amount of availble CPU memory.
 
+Warning: Collecting CPU metrics might affect performance. By default, CPU metrics are collected only if CPU(s) are used for inferencing. The collection of CPU metrics can be disabled by setting `cpu_monitor_disable` to `True`, see [Configuring Model Analyzer](docs/config.md) for details.
```
Comment (aramesh7) [inline docs/metrics.md:66]: "Warning:" looks better in bold italic _**Warning**_

**Review by aramesh7 — CHANGES_REQUESTED** [review]

File: docs/metrics.md (hunk 1 above)
Comment (kthui) [reply docs/metrics.md]: Updated.

File: docs/metrics.md [hunk 2]
```diff
@@ -63,6 +63,8 @@ recorded and aggregated over fixed intervals during a perf analyzer run.
 * `cpu_used_ram`: The total amount of memory used by all CPUs
 * `cpu_available_ram`: The total amount of availble CPU memory.
 
+**Warning**: Collecting CPU metrics might affect performance. By default, CPU metrics are not collected. To collect CPU metrics, add `cpu_used_ram` and/or `cpu_available_ram` to `metrics` flag, see [Configuring Model Analyzer](docs/config.md) for details.
```
Comment (aramesh7) [inline docs/metrics.md:66]: Maybe we can be specific and say that "Collecting CPU metrics might affect model inference metrics such as throughput and latency. "

**Review by aramesh7 — CHANGES_REQUESTED** [review]

File: docs/metrics.md (hunk 2 above)
Comment (kthui) [reply docs/metrics.md]: Updated.

**Review by aramesh7 — APPROVED** [review]

**Pull request merged**